editor/code_editor.cpp

```diff
@@ -222,6 +222,10 @@ void FindReplaceBar::_replace() {
 	} else {
 		text_editor->deselect(0);
 	}
+
+	if (!replace_text->has_focus()) {
+		text_editor->grab_focus();
+	}
```

Comment:
You should make sure that the "Replace" button isn't selected either, not just the LineEdit.